### DISCUSSION TITLE
mission planner to DJI waypoint

### DISCUSSION THREAD

**Discussion by superghm** [YarostheLaunchpadder/MissionPlanner-to-Litchi#33] [Q&A]
Hi! Very interesting tool. Thank you for develop it!
Is possible use this tool to export mission planner directly to DJI waypoints?

**Comment by YarosMallorca** [comment]
Not yet, I have the [DJI Mapper](https://github.com/YarosMallorca/DJI-Mapper) tool to create mappings for DJI Fly, check it out.
